rgsync/Connectors/sql_connectors.py

```diff
@@ -206,12 +205,11 @@ def __init__(self, connection, tableName, pk, exactlyOnceTableName=None):
 
     def PrepereQueries(self, mappings):
         def GetUpdateQuery(tableName, mappings, pk):
-            query = 'INSERT INTO %s' % tableName
+            query = 'REPLACE INTO %s' % tableName
             values = [val for kk, val in mappings.items() if not kk.startswith('_')]
-            values = [pk] + values
+            values = [self.pk] + values
```

Comment:
Whats the different between self.pk and pk?

Reply:
I was going to ask you that. We're not consistent